templates/nextflow-ecs-cluster.yaml

```diff
@@ -140,7 +158,10 @@ Resources:
       LaunchConfigurationName: !Ref EcsLaunchConfiguration
       MinSize: '0'
       MaxSize: !Ref AutoscalingGroupMaxSize
-      DesiredCapacity: !Ref AutoscalingGroupMaxSize
+      DesiredCapacity: !Ref AutoscalingGroupDesiredCapacity
+      TerminationPolicies:
+        - OldestLaunchConfiguration
+        - OldestInstance
```

Comment:
any reason why not just let AWS manage this with `Default` setting?

Reply:
Yes, I want to be explicit about how this is going to behave so that anyone supporting it knows what to expect. It's also a little different from the default behavior. You can look that up.